src/content/docs/reference/attestations.mdx

```diff
@@ -1,35 +1,56 @@
+All attestations using this schema are displayed under the **Propdates** section of the proposal UI. It supports Markdown, IPFS links, structured JSON, and threaded discussions via reply references.
+
+## Sequence Diagram
+
+![EAS Attestaion Flow](./public/guides/propdateGuide/attestation-flow.png)
+
+```mermaid
+sequenceDiagram
+    participant User
+    participant DAO Frontend
+    participant Ethereum Attestation Service (EAS)
+    participant Blockchain
+
+    User->>DAO Frontend: Compose Propdate update
+    DAO Frontend->>User: Prompt to sign transaction
+    User->>Blockchain: Sign and submit Propdate attestation
+    Blockchain->>EAS: Store attestation
+    DAO Frontend->>EAS: Fetch Propdates for proposal/activity feed
+    EAS->>DAO Frontend: Return Propdate data
+    DAO Frontend->>User: Display Propdates (filtered by member/all)
+```
 
 ## Schema Specification
 
-The schema with the UID `0x8bd0d42901ce3cd9898dbea6ae2fbf1e796ef0923e7cbb0a1cecac2e42d47cb3` enables Nouns Builder Propdates. It is deployed on-chain on all networks supported by Nouns Builder, consisting of mandatory and optional fields. The payload displayed on the frontend is wrapped in `message`, which MAY contain a *JSON* string.
+The schema with the UID `0x8bd0d42901ce3cd9898dbea6ae2fbf1e796ef0923e7cbb0a1cecac2e42d47cb3` enables Nouns Builder Propdates. It is deployed on-chain on all networks supported by Nouns Builder, **except ZORA** consisting of mandatory and optional fields. The payload dispalyed on the frontend is wrapped in `message`, which MAY contain a *JSON* string.
 
 ### Deployed Schemas
 
 - [Base](https://base.easscan.org/schema/view/0x8bd0d42901ce3cd9898dbea6ae2fbf1e796ef0923e7cbb0a1cecac2e42d47cb3),
-- [Zora](),
+- **EAS, hence Nouns Builder Propdates are not available on Zora Mainnet.**
 - [OP Mainnet](https://optimism.easscan.org/schema/view/0x8bd0d42901ce3cd9898dbea6ae2fbf1e796ef0923e7cbb0a1cecac2e42d47cb3),
 - [Ethereum Mainnet](https://easscan.org/schema/view/0x8bd0d42901ce3cd9898dbea6ae2fbf1e796ef0923e7cbb0a1cecac2e42d47cb3),
 - [Base Sepolia](https://base-sepolia.easscan.org/schema/view/0x8bd0d42901ce3cd9898dbea6ae2fbf1e796ef0923e7cbb0a1cecac2e42d47cb3),
 - [OP Sepolia](https://optimism-sepolia.easscan.org/schema/view/0x8bd0d42901ce3cd9898dbea6ae2fbf1e796ef0923e7cbb0a1cecac2e42d47cb3),
-- [Zora Sepolia](),
+- **EAS, hence Nouns Builder Propdates are not available on Zora Sepolia.**
 - [Ethereum Sepolia](https://sepolia.easscan.org/schema/view/0x8bd0d42901ce3cd9898dbea6ae2fbf1e796ef0923e7cbb0a1cecac2e42d47cb3)
 
 ### Schema Structure
 
-```
-bytes32 proposalId;
-bytes32 originalMessageId;
-uint8 messageType;  // 0: INLINE_TEXT, 1: INLINE_JSON, 2: URL_TEXT, 3: URL_JSON
-string message;     // Inline content or off-chain URL
+```solidity
```

Comment:
@benedictvscriticus if looks like you reverted the changes I had made to the schema. The name of the variables is important as it gets cached when deploying the schema on-chain.

It should be ```
bytes32 proposalId;
bytes32 originalMessageId;
uint8 messageType;  // 0: INLINE_TEXT, 1: INLINE_JSON, 2: URL_TEXT, 3: URL_JSON
string message;     // Inline content or off-chain URL
```solidity

Reply:
fixed in a11858d1719ddd16f4e5ece2590f6c45f4fbfa28